### ISSUE TITLE
Unordered list markdown syntax is incorrect

### ISSUE THREAD

**Issue by naktinis** [adbar/trafilatura#600]
It appears that `-` is inserted after the line instead of before.

Version tested: `1.9.0`.

Example:
```
html_list = """
<html>
<article>
<ul>
  <li>One</li>
  <li>Two</li>
  <li>Three</li>
  <li>Four</li>
</ul>
</article>
</html>
"""
print(extract(html_list, output_format='txt'))
```

Output:
```
One
-
Two
-
Three
-
Four
-
```

Expected output:
```
- One
- Two
- Three
- Four
```

**Comment by johnnymakes** [comment]
+1

**Comment by adbar** [comment]
This has been solved in #598.

**Comment by adbar** [comment]
To be clear: you can use the latest version straight from the repository or wait for the next release (happening soon).
